source/trunk/Esf.DataAccess.Tests/Esf.DataAccess.Tests.csproj

```diff
@@ -0,0 +1,88 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we should group together test projects with the functionality projects: it: in folder Esf.DataAccess projects Esf.DataAccess and Esf.DataAccess.Tests